tests/test_models.py

```diff
@@ -315,3 +327,34 @@ def test_zigzag_produces_warning(self):
         for engine in stdpopsim.all_engines():
             with self.assertWarnsRegex(UserWarning, "Zigzag_1S14"):
                 engine.simulate(model, contig, samples, dry_run=True)
+
+
+class TestMutationRates(unittest.TestCase):
+    def test_mutation_rate_warning(self):
+        species = stdpopsim.get_species("HomSap")
+        model = species.get_demographic_model("OutOfAfrica_3G09")
```

Comment:
Better do a copy.deepcopy() on this object, as you're potentially modifying the global state and this might affect other test. Ditto below.